src/rules/orderedImportsRule.ts

```diff
@@ -51,16 +60,23 @@ export class Rule extends Lint.Rules.AbstractRule {
                     type: "string",
                     enum: ["case-insensitive", "lowercase-first", "lowercase-last"],
                 },
+                "source-imports-order": {
+                    type: "string",
+                    enum: ["case-insensitive", "lowercase-first", "lowercase-last"],
+                },
             },
             additionalProperties: false,
         },
-        optionExamples: ["true", '[true, {"named-imports-order": "lowercase-first"}]'],
+        optionExamples: [
+            "true",
+            '[true, {"named-imports-order": "lowercase-first", "source-imports-order": "lowercase-last"}]',
```

Comment:
I'd be fine with separate options. Module names tend to be more uniform (mostly lowercase, hyphenated) so I doubt this setting will matter much either way.